src/lib/components/ui/ArticleList.svelte

```
@@ -14,12 +14,14 @@
 
 	const {
 		articles,
+		articleCategories = [],
```

ESLint

[error] 'articleCategories' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)